docs/_includes/getting-started.html

```diff
@@ -2,29 +2,7 @@
     <div class="centered" style="padding-top: 32px">
         <h1 id="getting-started">Getting Started</h1>
         <p>
-            Using Dotty to compile your project is now pretty simple. Create the
-            following structure:
-        </p>
-        <pre class="sourceCode bordered">
-<code>.
-├── build.sbt
-├── project
-│   ├── build.properties
-│   └── plugins.sbt
-└── src</code></pre>
-        <h2 id="build.sbt">build.sbt</h2>
-        <pre class="sourceCode bordered"><code>name := "application"
-version := "0.1"
-enablePlugins(DottyPlugin)</code></pre>
-        <h2 id="build.properties">build.properties</h2>
-        <pre class="sourceCode bordered"><code>sbt.version=0.13.11</code></pre>
-        <h2 id="plugins.sbt">plugins.sbt</h2>
-        <pre class="sourceCode bordered"><code>addSbtPlugin("com.felixmulder" % "sbt-dotty" % "0.1.7")</code></pre>
-        <p>
-            This plugin is based on the
-            <a href="https://github.com/smarter/dotty-example-project">
-                dotty-example-project
-            </a>, but let’s you skip building dotty from scratch.
+            See the <a href="https://github.com/lampepfl/dotty-example-project">example project</a>.
```

Comment:
> Maybe we should restructure the intro page and put this information at the top after the logo and show a condensed version of the dotty example project README which we also refer to for more information.

I really don't want to duplicate information, we're already doing a bad job at keeping one copy of our documentation up to date, and we completely fail for information in multiple places (the homepage talked about a very old version of the sbt-dotty plugin).